hack/common

```diff
@@ -165,10 +165,16 @@ function login_to_registry() {
         fi
     fi
     podman login --tls-verify=false -u "$username" -p "$token" "$1" > /dev/null
```

Comment:
it works if you use docker instead:
```bash
docker login -u "$username" -p "$token" "$1" > /dev/null
```
I guess we just can't use podman yet :-(

Reply:
So please revise this PR to only change the single line above

Reply:
Is you suggestion replacing
`podman login --tls-verify=false -u "$username" -p "$token" "$1" > /dev/null`
with
`docker login -u "$username" -p "$token" "$1" > /dev/null`
?  And reverting the rest?  If yes, let me try the changes in my next deploy.  I've been using this patch for my .local CLO debugging... :)

Reply:
> Is you suggestion replacing

yes - see https://github.com/openshift/origin-aggregated-logging/pull/1771

Reply:
Cool!  It worked.  Let me update this PR...